lib/sproxyd.js

```diff
@@ -151,8 +151,18 @@ class SproxydClient {
         let counter = tries;
         log.info('request', { method, value, args, counter });
 
+        let getResponse = false;
         this._handleRequest(method, stream, size, key, log, (err, ret) => {
             if (err && !err.isExpected) {
+                if (getResponse === true) {
+                    const error = new Error('trying to put data after sproxyd' +
+                        ' send back a response, size of the object is may ' +
```

Comment:
Typo:   s/ is//
Suggestion: Couldn't we measure actual size and here dump actual size + expected size  as additional data for investigation ?